tests/compilation/passingContracts.ts

```diff
@@ -1,5 +1,6 @@
 export const passingContracts = [
-  'tests/compilation/contracts/ERC20.sol',
+  // FIXME: uncomment, when storage is ready for Cairo 1
+  // 'tests/compilation/contracts/ERC20.sol',
```

Comment:
Why the PR breaks compilation of ERC20?

Reply:
The storage system is not reworked yet, so storage functions have incorrect types.